addons/purchase/views/res_partner_views.xml

```diff
@@ -52,7 +52,12 @@
             <field name="inherit_id" ref="base.res_partner_kanban_view"/>
             <field name="arch" type="xml">
                 <xpath expr="//footer/div" position="inside">
-                    <a t-if="record.purchase_order_count.value>0" type="action" name="%(purchase.act_res_partner_2_purchase_order)d" class="btn btn-sm btn-link smaller" role="button">
+                    <a t-if="record.purchase_order_count?.value"
+                        class="btn btn-sm btn-link smaller"
+                        groups="purchase.group_purchase_user"
+                        name="%(purchase.act_res_partner_2_purchase_order)d"
+                        role="button"
+                        type="action">
```

Comment:
Now that the counts are 0 if you don't have the group, why do you need to change the views ?

Reply:
Mhm, ok then :ok_hand: